Aztec/Classes/Public/TextKit/TextList.swift

```diff
@@ -0,0 +1,67 @@
+import Foundation
+import UIKit
+
+
+// MARK: - Text List
+//
+class TextList
+{
+    // MARK: - Nested Types
+
+    /// Kind of Lists
+    ///
+    enum Kind {
+        case Ordered
+        case Unordered
+    }
+
+    // MARK: - Properties
+
+    /// Kind of List: Ordered / Unordered
+    ///
+    let kind: Kind
```

Comment:
Definitely agree on this one. The `Kind` enum was, originally, called `Type`. 
But i've opted to rename it, since it collides with Swift's `Type` [reserved keyword](https://developer.apple.com/library/content/documentation/Swift/Conceptual/Swift_Programming_Language/Types.html).

Thanks for the heads up!!